data.sql

```diff
@@ -42,4 +42,44 @@ UPDATE animals SET owner_id = 1 WHERE name = 'Agumon';
 UPDATE animals SET owner_id = 2 WHERE name IN( 'Gabumon', 'Pikachu');
 UPDATE animals SET owner_id = 3 WHERE name IN( 'Devimon', 'Plantmon');
 UPDATE animals SET owner_id = 4 WHERE name IN( 'Charmander', 'Squirtle', 'Blossom');
-UPDATE animals SET owner_id = 5 WHERE name IN( 'Angemon', 'Boarmon');
\ No newline at end of file
+UPDATE animals SET owner_id = 5 WHERE name IN( 'Angemon', 'Boarmon');
```

Comment:
- [ ] [OPTIONAL] Please don't forget to add an MIT license to your project

- [ ] [OPTIONAL] I would also suggest adding at least 5 commits to your pull requests to make it professional 💎